spec/web_helpers.rb

```diff
@@ -9,6 +9,15 @@ def sign_up
   click_button 'Sign up'
 end
 
+def sign_up_2
```

Comment:
Looks like duplication. You could make the original sign_up method accept an argument for email and/or password.